gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -133,8 +158,16 @@ def get_export_resources(
     )
     validate_release_ht = PipelineStepResourceCollection(
         "--validate-release-ht",
-        input_resources={"create_release_sites_ht.py": {"release_ht": release_sites()}},
-        output_resources={"validated_ht": validated_release_ht(test=test)},
+        input_resources={
+            "create_release_sites_ht.py": {
+                "release_ht": hl.read_table(
+                    "gs://gnomad-tmp/gnomad.exomes.v4.0.qc_data/release/gnomad.genomes.sites.test.updated_101923.ht"
+                )  # release_sites(data_type=data_type) #TODO: Restore once PR is ready for merge
+            }
+        },
+        output_resources={
+            "validated_ht": validated_release_ht(test=test, data_type=data_type)
```

Comment:
I just noticed `validated_release_ht` has an overwrite parameter that can me removed